packages/web-components/src/components/logo-grid/logo-grid.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { css, customElement, html, property } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './logo-grid.scss';
+import DDSContentBlock from '../content-block/content-block';
+import '../content-block/content-block-heading';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Logo grid.
+ *
+ * @element dds-logo-grid
+ */
+@customElement(`${ddsPrefix}-logo-grid`)
+class DDSLogoGrid extends StableSelectorMixin(DDSContentBlock) {
+  @property({ attribute: 'hide-border', reflect: true, type: Boolean })
+  hideBorder = false;
+
+  protected updated() {
+    this.toggleAttribute('hide-border', this.hideBorder);
+  }
```

Comment:
`@property({ attribute: 'hide-border', reflect: true, type: Boolean })` takes care of this automatically.


```suggestion
```

Reply:
@RaphaelAmadeu Have you tried to removing this method? Thanks!